qa/sc_bwt_minimum_value.py

```diff
@@ -0,0 +1,393 @@
+        print("Withdrawal epoch 0 certificate hex = " + we0_cert_hex)
+        assert_equal(self.sc_nodes_bootstrap_info.sidechain_id, we0_cert["cert"]["scid"], "Sidechain Id in certificate is wrong.")
+        assert_equal(0, we0_cert["cert"]["epochNumber"], "Sidechain epoch number in certificate is wrong.")
+        assert_equal(we0_end_epoch_cum_sc_tx_comm_tree_root, we0_cert["cert"]["endEpochCumScTxCommTreeRoot"],
+                     "Sidechain endEpochCumScTxCommTreeRoot in certificate is wrong.")
+        assert_equal(0, we0_cert["cert"]["totalAmount"], "Sidechain total amount in certificate is wrong.")
+
+        # Generate MC block and verify that certificate is present
+        we1_2_mcblock_hash = mc_node.generate(1)[0]
+        assert_equal(0, mc_node.getmempoolinfo()["size"], "Certificate expected to be removed from MC node mempool.")
+        assert_equal(1, len(mc_node.getblock(we1_2_mcblock_hash)["tx"]), "MC block expected to contain 1 transaction.")
+        assert_equal(1, len(mc_node.getblock(we1_2_mcblock_hash)["cert"]), "MC block expected to contain 1 Certificate.")
+        assert_equal(we0_certHash, mc_node.getblock(we1_2_mcblock_hash)["cert"][0], "MC block expected to contain certificate.")
+        print("MC block with withdrawal certificate for epoch 0 = {0}\n".format(str(mc_node.getblock(we1_2_mcblock_hash, False))))
+
+        # Generate SC block and verify that certificate is synced back
+        scblock_id4 = generate_next_blocks(sc_node, "first node", 1)[0]
+        check_mcreference_presence(we1_2_mcblock_hash, scblock_id4, sc_node)
+
+        # Check that certificate generation skipped because chain have certificate with same quality
+        time.sleep(2)
+        assert_false(sc_node.submitter_isCertGenerationActive()["result"]["state"], "Expected certificate generation will be skipped.")
+
+        # Verify Certificate for epoch 0 on SC side
+        mbrefdata = sc_node.block_best()["result"]["block"]["mainchainBlockReferencesData"][0]
+        we0_sc_cert = mbrefdata["topQualityCertificate"]
+        assert_equal(len(mbrefdata["lowerCertificateLeaves"]), 0)
+        assert_equal(self.sc_nodes_bootstrap_info.sidechain_id, we0_sc_cert["sidechainId"],
+                     "Sidechain Id in certificate is wrong.")
+        assert_equal(0, we0_sc_cert["epochNumber"], "Sidechain epoch number in certificate is wrong.")
+        assert_equal(we0_end_epoch_cum_sc_tx_comm_tree_root, we0_sc_cert["endCumulativeScTxCommitmentTreeRoot"],
+                     "Sidechain endEpochCumScTxCommTreeRoot in certificate is wrong.")
+        assert_equal(0, len(we0_sc_cert["backwardTransferOutputs"]), "Backward transfer amount in certificate is wrong.")
+        assert_equal(we0_certHash, we0_sc_cert["hash"], "Certificate hash is different to the one in MC.")
+
+
+        # Checking withdrawCoins
+        # Try to withdraw coins from SC to MC: amount below the dust threshold
+        mc_address1 = self.nodes[0].getnewaddress()
```

Comment:
Test flow until this point is a duplication of `sc_backward_transfer.py`
What is the sense to test Cert with no BTs here?
Please, remove the redundant part and keep only the cases of BT with minimum amount.
Also, update the test description.

Reply:
Done.